### PR TITLE
Add v2 call

### PR DESCRIPTION
**Pull request by jcorbin**

### CODE REVIEW

**Comment by jcorbin** [comment]
don't review quite yet, going to PR header and checksum separately firt

**Comment by jcorbin** [comment]
Okay ready for review @Raynos @kriskowal

**Comment by kriskowal** [comment]
This looks good to me. Note to self to verify that there’s good coverage on the read and write composition functions.

File: v2/call.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    Checksum.read, // csumtype:1 (csum:4){0,1}
+    read.buf2,     // arg1~2
+    read.buf2,     // arg2~2
+    read.buf2      // arg3~2
+]), function buildCallRes(results, buffer, offset) {
+    var flags = results[0];
+    if (flags & CallResponse.Flags.Fragment) {
+        throw new Error('streaming call request not implemented');
+    }
+    var code = results[1];
```
Comment (Raynos) [inline v2/call.js]: can we check that this is either 0 or 1 ?

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**